sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/DataProcessor.cs

```diff
@@ -74,7 +74,7 @@ public TestRunShardDto GetTestRunShard()
                     Status = "RUNNING",
                     StartTime = startTime,
                 },
-                Workers = 1
+                Workers = _cloudRunMetadata.NumberOfTestWorkers
```

Comment:
Q: I assume this is user configured worker count which could be different from affective parallelism and this behavior is same in JS as well.

Reply:
yes, this is the configured worker count